src/Nokia_LCD.cpp

```diff
@@ -86,6 +91,9 @@ void Nokia_LCD::begin() {
         pinMode(kBl_pin, OUTPUT);
     }
 
+    // Sets the default font
+    setFont(&defaultFont);
+
```

Comment:
We can set the font to the default value in the constructor's initializer list.
E.g. `kUsingHardwareSPI{true}, customFont{&defaultFont}`

Reply:
Great tip, done